pages/common/ln.md

```diff
@@ -4,16 +4,16 @@
 
 - create a symbolic link to a file or folder
 
-`ln -s {{path/to/original/file}} {{path/to/link}}`
+`ln -s {{path/to/target}} {{path/to/link}}`
 
 - overwrite a symbolic link to a file
 
-`ln -sf {{path/to/new/original/file}} {{path/to/file/link}}`
+`ln -sf {{path/to/new/target/file}} {{path/to/file/link}}`
 
 - overwrite a symbolic link to a folder
 
-`ln -sfT {{path/to/new/original/file}} {{path/to/folder/link}}`
+`ln -sfT {{path/to/new/target/folder/}} {{path/to/folder/link}}`
```

Comment:
Overwriting a symbolic link to a folder doesn't seem like a common thing to do, IMHO. Should we just drop this example?